plugins/faustwp/includes/replacement/graphql-callbacks.php

```diff
@@ -59,5 +59,11 @@ function url_replace_recursive( &$data ) {
 		} elseif ( is_array( $value ) ) {
 			url_replace_recursive( $value );
 		}
+
+		if ( 'path' === $key && is_multisite() ) {
```

Comment:
Would moving this condition up into the existing `if elseif` improve readability? I find it helpful with recursion to try and keep the recursive case as the final statement in the function.

It might also be good to keep an `is_string()` check here in case plugins extend the schema with some kind of oddball value for a field named `path`.